daemon/mgr/network.go

```diff
@@ -537,18 +536,16 @@ func (nm *NetworkManager) getNetworkSandbox(id string) libnetwork.Sandbox {
 	return sb
 }
 
-// libnetwork's logrus version is different from pouchd,
-// so we need to set libnetwork's logrus addintionly.
 func initNetworkLog(cfg *config.Config) {
```

Comment:
If pouchd use the lastest libnetwork, initialize logrus is unuse.

Reply:
Thank you, I thought about this but was not sure then.